code/modules/events/profound_peace.dm

```diff
@@ -0,0 +1,51 @@
+var/list/forms_of_nirvana = list("buddha","chill")
+/datum/event/profound_peace
+	announceWhen	= 1
+	startWhen = 1
+	endWhen			= 120 //4 minutes
+	var/list/participants = list()
+	var/mode
+
+/datum/event/profound_peace/can_start(var/list/active_with_role)
+	return 20 * forms_of_nirvana.len
+
+/datum/event/profound_peace/announce()
+	command_alert(/datum/command_alert/nirvana)
+
+/datum/event/profound_peace/setup()
+	mode = pick_n_take(forms_of_nirvana)
+	if(!mode)
+		kill()
+		qdel(src)
+	for(var/mob/living/carbon/human/H in player_list)
```

Comment:
A miner on the roid can return to the main station without being included in the participants list if it is timed right - is this intentional?

Reply:
peace achieved between beno and spess kind, billions must live

Reply:
what's going to stop a scientist deciding to commit mass murder the second it begins, scientists have lots of reasons to leave the asteroid base at any moment - so it won't be suspicious at all when the scientist antag leaps out of staring at a wall blankly and goes to mass murder the station

Reply:
In the situation of an antag scientist who has previously armed himself with valid tools, yeah, he would have 4 minutes to try to quickly cause chaos. That's a good thing! The crew aren't helpless, as Schlomo has demonstrated.

Re:nodeath mode - that's just how Buddha mode works. It's very interesting in practice and if you haven't seen it in gameplay, I'd say you just have to trust me that it's cool. I use it a lot when bussing.

Reply:
I'm constantly amused by Schlomo becoming a balancing factor/example.